common/serialization/subtitles/subtitles.cpp

```diff
@@ -278,7 +278,10 @@ void parse_subtitle(const goos::Object& data,
         scene.m_sorting_group_idx = db.m_subtitle_groups->find_group_index(scene.m_sorting_group);
 
         for_each_in_list(entries, [&](const goos::Object& entry) {
-          if (entry.is_pair()) {
+          if (entry.is_empty_list()) {
+            lg::error("Empty list found in scene {}; skip it", scene.name());
+            return;
```

Comment:
Under what circumstance is an empty list being generated inside a subtitle scene? Seems very error prone.

Reply:
Can you not just generate nothing at all instead? I don't recall there being a requirement for a scene to have lines.

Reply:
> Can you not just generate nothing at all instead? I don't recall there being a requirement for a scene to have lines.

https://github.com/open-goal/jak-project/pull/1516#discussion_r905396698 was a reply to the wrong thing. it was meant to be a reply to this. anyway, i thought it wasn't choking on scenes with no lines, but actually when it's a movie and not a hint, i get `invalid pair` in a `car`. i've added a check for that as well.

Reply:
Wait, so what's generated when that happens? Can you post it?